ts/features/idpay/initiative/configuration/screens/ConfigurationSuccessScreen.tsx

```diff
@@ -24,31 +34,73 @@ const ConfigurationSuccessScreen = () => {
   const { initiativeName } = initiativeDetails;
 
   const handleNavigateToInitiativePress = () => {
-    send({ type: "COMPLETE_CONFIGURATION" });
+    configurationMachine.send({ type: "COMPLETE_CONFIGURATION" });
+  };
+
+  const handleAddPaymentMethodButtonPress = () => {
```

Comment:
Maybe this braces {} could be removed.

Reply:
Done in https://github.com/pagopa/io-app/pull/4290/commits/54f91ba77a77c9b030a4f37e4b6575d1d80b77e1